app/conf/assets.conf

```diff
@@ -125,7 +125,7 @@ packages = {
 	openlayers = {
 		maps = OpenLayers.js,
 		mobile = OpenLayers.mobile.js,
-		stamen = http://maps.stamen.com/js/tile.stamen.js?v1.1.3
+		stamen = https://stamen-maps.a.ssl.fastly.net/js/tile.stamen.js
```

Comment:
Protocol-less doesn't work because the AssetManager class has some logic that looks for http(s?) and treats it as local otherwise. 

The version specifier doesn't do anything anyway, go to their old url (the http one with the version) and it gets 1.3.0 not 1.1.3.